extensions/wikia/CuratedContent/CuratedContentHelper.class.php

```diff
@@ -54,12 +53,7 @@ public function fillItemInfo( &$item ) {
 
 				case self::STR_CATEGORY:
 					$category = Category::newFromTitle( $title );
-					if ( $category instanceof Category && $category->getID() ) {
-						$count = $category->getPageCount();
-						if ( empty( $count ) ) {
-							$item['type'] = self::STR_EMPTY_CATEGORY;
-						}
-					} else {
+					if ( !( $category instanceof Category && $category->getID() ) ) {
```

Comment:
So you are setting 'type' to null for empty categories? If so, it won't work.
Take this category for instance: http://concf.wikia.com/wiki/Category:Lorem. It's wgArticleId is 0 and it is an instance of a Category class and it has some items - so it's valid candidate for Curated Content but you would set 'type' to null